jetty-util-ajax/src/main/java/org/eclipse/jetty/util/ajax/AsyncJSON.java

```diff
@@ -1115,6 +1115,20 @@ protected RuntimeException newInvalidJSON(ByteBuffer buffer, String message)
         return new IllegalArgumentException(builder.toString());
     }
 
+    private static boolean isWhitespace(byte ws)
+    {
+        switch (ws)
+        {
+            case ' ':
+            case '\n':
+            case '\r':
+            case '\t':
+                return true;
+            default:
+                return false;
```

Comment:
This whitespace list seems awfully narrow.

Reply:
Here's mozilla's take on whitespace - https://developer.mozilla.org/en-US/docs/Glossary/whitespace

Reply:
All the JSON specs agree on those 4 chars only.
https://tools.ietf.org/html/rfc7159#section-2

Reply:
The JSON RFC clearly states that only those four characters are to be considered whitespaces: https://tools.ietf.org/html/rfc8259 (top of page 5).

Reply:
okay.